source/documentation/standards/hosting.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: How to host services
+last_reviewed_on: 2020-09-01
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+At MOJ you should use the [Cloud
+Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) to
+host your service unless you're advised otherwise by a member of [the
+hosting team](https://peoplefinder.service.gov.uk/teams/hosting). Your
+team can get started on the Cloud Platform without letting us know;
```

Comment:
This should probably be more explicit to who "us" is - Hosting team, or Cloud Platform team?